### PR TITLE
0.30 docs

### LINKED ISSUE #7995: Document new features and changes in 0.30
- [ ] Changes to collections:
  - [ ] Overworld
  - [ ] Data browsing
  - [ ] Creating new collections, dashboards, pulses
  - [ ] Sub-collections
  - [ ] Pinning
  - [ ] Personal collections
  - [ ] Updated permissions grid
- [ ] Search
- [ ] Conditional formatting for tables
- [ ] Comparison x-rays
- [ ] Adding available fields to a table

Anything else I'm forgetting?

### PR DESCRIPTION
**Pull request by mazameli**
Still need to:
- [x] add in new screenshots
- [x] update/remove old screenshots

Has docs for:
- [x] Adding available fields to a table
- [x] Changes to collections:
  - [x] Overworld
  - [x] Data browsing
  - [x] Creating new collections, dashboards, pulses
  - [x] Sub-collections
  - [x] Pinning
  - [x] Personal collections
  - [x] Updated permissions grid
- [x] Search
- [x] Conditional formatting for tables
- [x] Comparison x-rays

Resolves #7995.

### CODE REVIEW

File: docs/administration-guide/welcome-to-metabase.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Welcome to Metabase!
```
Comment (salsakran) [inline docs/administration-guide/welcome-to-metabase.md]: should this be in the admin guide?

Comment (mazameli) [reply docs/administration-guide/welcome-to-metabase.md]: Yeah, I initially put this in the admin guide, then decided it was better as an additional to the user guide's "what is metabase" page so I put it in there, but forgot to remove it from the admin guide. I'll remove it.

**Comment by mazameli** [comment]
Okay, this should be ready to go now. You'll notice I pulled in master so that I could take screenshots of my local instance without switching branches, hence the additional commits and yarn prettier etc.

@kdoh I added you as a second reviewer because there are quite a bit of changes here and another pair of eyes would be welcome.

File: docs/administration-guide/06-collections.md [hunk 2]
```diff
@@ -1,35 +1,61 @@
 ## Creating Collections for Your Saved Questions
 ---
 
-Collections are a great way to organize your saved questions and decide who gets to see and edit things. Collections could be things like, "Important Metrics," "Marketing KPIs," or "Questions about users." Multiple [user groups](05-setting-permissions.md) can be given access to the same collections, so we don't necessarily recommend naming collections after user groups.
+![Collection detail](images/collections/collection-detail.png)
+
+Collections are a great way to organize your saved questions and decide who gets to see and edit things. Collections could be things like, "Important Metrics," "Product Team," "Marketing KPIs," or "Questions about users." Collections can even contain other collections, allowing you to create an organizational structure that fits your team. You can also choose which user groups should have what level of access to your collections (more on that below).
```
Comment (kdoh) [inline docs/administration-guide/06-collections.md]: We should probably update this first sentence to reflect the fact that you can organize more than just questions in collections.

File: docs/administration-guide/06-collections.md [hunk 3]
```diff
@@ -1,35 +1,61 @@
...
+If a user has Curate access for a collection, they can create new sub-collections inside it and edit the contents of the collection. From the detail view of any collection, click on the `Create a collection` button to make a new one. Give your collection a name, choose where it should live, and give it a description if you'd like.
+
+![Create collection](images/collections/create-collection.png)
+
+By default, new collections will have the same permissions settings as the collection it was created in (its "parent" collection), but you can change those settings from the Edit menu.
+
+### Pinning things in collections
+![Pins](images/collections/pinned-items.png)
+
+One great new feature in Metabase is that you can now pin the most important couple of items in each of your collections to the top. Pinning a dashboard or question turns it into a big, eye-catching card that will help make sure that folks who are browsing your Metabase instance will always know what's most important.
```
Comment (kdoh) [inline docs/administration-guide/06-collections.md]: Dunno if we should describe things as a "new" feature in the docs just to make these a bit more evergreen.

File: docs/users-guide/01-what-is-metabase.md [hunk 4]
```diff
@@ -1,6 +1,54 @@
...
+
+### Browse your data
+
+![Browse data](./images/browse-data.png)
+
+If you connected your database to Metabase during setup, you'll see it listed at the bottom of the homepage along with the sample dataset that Metabase comes with. Click on a database to see its contents. You can click on a table to see its rows, or you can also click on the bolt icon to x-ray a table and see an automatic exploration of it, or click on the book icon to go to the data reference view for that table to learn more about it.
+
+### Explore your analytics
+
+As long as you're not the first Metabase user on your team, you can browse the dashboards, questions, and collections your teammates have made. [More about exploring](03-basic-exploration.md)
```
Comment (kdoh) [inline docs/users-guide/01-what-is-metabase.md]: This first sentence is a bit confusing. I guess its reflecting the fact there might not be stuff on a new instance but not sure we need to call that out explicitly.

**Comment by mazameli** [comment]
Thanks for the comments, @kdoh. Made some changes to address those items. Unless anyone wants to take another look, seems like this PR is ready to go.

**Comment by salsakran** [comment]
are there any screenshots in the current docs that should be updated?

**Comment by mazameli** [comment]
Yes, I updated quite a number of them.

File: docs/administration-guide/06-collections.md [hunk 5]
```diff
@@ -1,35 +1,61 @@
...
+- **No access:** the user won't see this collection listed, and doesn't have access to any of the items saved within it.
 
-You'll see a table with your user groups along the top and all your collections down along the left. A user group can have View access, Curate access, or no access to a given collection.
+![Permissions](images/collections/collection-permissions.png)
 
-- View access: can see all the questions in the collection, **even if the user doesn't have access to the underlying data used to create the question.**
-- Curate access: can additionally move questions in or out of the collection, and edit the questions in the collection.
-- No access: won't see the collection listed on the Questions page, and can't see questions from this collection in dashboards or when creating a Pulse.
+If you want to see the bigger picture of what permissions your user groups have for all your collections, jut click the link that says `See all collection permissions`. You'll see a table with your user groups along the top and all your collections down along the left. Click the `View collections` link under any collection that contains more collections to zoom in and see its contents:
```
Comment (salsakran) [inline docs/administration-guide/06-collections.md]: `jut` -> `just`

Comment (mazameli) [reply docs/administration-guide/06-collections.md]: Thanks, fixed.

**Pull request merged**